boilerplate/openshift/golang-osd-operator/standard.mk

```
@@ -89,6 +91,7 @@ push: docker-push
 go-check: ## Golang linting and other static analysis
 	${CONVENTION_DIR}/ensure.sh golangci-lint
 	GOLANGCI_LINT_CACHE=${GOLANGCI_LINT_CACHE} golangci-lint run -c ${CONVENTION_DIR}/golangci.yml ./...
+	test "${GOLANGCI_OPTIONAL_CONFIG}" = "" || test ! -e "${GOLANGCI_OPTIONAL_CONFIG}" || GOLANGCI_LINT_CACHE="${GOLANGCI_LINT_CACHE}" golangci-lint run -c "${GOLANGCI_OPTIONAL_CONFIG}" ./...
```

This looks right.

Maybe it's just me, but I find the logic confusing as written. To me it would be more intuitive as a chain of positive assertions:
```suggestion
	test -n "${GOLANGCI_OPTIONAL_CONFIG}" && test -f "${GOLANGCI_OPTIONAL_CONFIG}" && GOLANGCI_LINT_CACHE="${GOLANGCI_LINT_CACHE}" golangci-lint run -c "${GOLANGCI_OPTIONAL_CONFIG}" ./...
```

---

It would definitely be much more intuitive but unfortunately the target will return `1` from the target if `GOLANGCI_OPTIONAL_CONFIG` is not defined or doesn't exist :(

---

Oh, yeah.